index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Home Page</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <ul class="navbar-items">
+                <li class="logo"> <h3>Welcome</h3></li>
+                <li class="menu-button"><img src="images/IconMenu.jpg" alt=""></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <h1>Hey there. I’m Juan Fco. <br> <span>I’m a software developer</span></h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <ul class="contact-info">
+                <li> <a href="https://twitter.com/jrosario19" class="contact-info-button"><img src="images/twitter.svg" alt=""></a> </li>
+                <li> <a href="https://linkedin.com/in/juan-francisco-rosario-suli-44595051" class="contact-info-button"><img src="images/Linkedinicon.svg" alt=""></a> </li>
+                <li> <a href="#" class="contact-info-button"><img src="images/medium.svg" alt=""></a> </li>
+                <li> <a href="https://github.com/jrosario19" class="contact-info-button"><img src="images/github.svg" alt=""></a> </li>
+                <li> <a href="#" class="contact-info-button"><img src="images/angellist.svg" alt=""></a> </li>
+            </ul>                
```

Comment:
- [ ] Kindly add this button at the end of your headline section, to match the Figma design.

This one
![image](https://user-images.githubusercontent.com/35535130/158706438-d93fbcba-9a5b-41b3-b994-ec14f646c112.png)